.mci.yml

```diff
@@ -590,7 +589,7 @@ functions:
                           echo "CRYPT_SHARED_LIB_PATH=$CRYPT_SHARED_LIB_PATH"
                       fi
 
-                      # Run tests and examples 1-by-1 with "test_params" so we can run them with valgrind.
+                      # Run tests and examples 1-by-1 with "test_params" so we can run them with asan/ubsan.
```

Comment:
I think comment is no longer relevant and can be removed.

`valgrind_test_params` included the `valgrind` command. `asan_test_params` and `ubsan_test_params` only include environment variables.

I do not think it is necessary to run the tests individually for `asan/ubsan`. I suggest removing this comment.